doc/chomp_interface/chomp_interface_tutorial.rst

```diff
@@ -14,7 +14,7 @@ You should also have gone through the steps in `Visualization with MoveIt! RViz
 
 Prerequisites
 --------------
- 1. You must have the latest version of MoveIt! installed. On ROS Kinetic you will need to build MoveIt! from source. We will go through the steps for doing this below.
+ 1. You must have the latest version of MoveIt! installed. On ROS Kinetic you will need to build MoveIt! from source. A build from source is required as CHOMP is not officially released so ``apt-get install`` for moveIt would not be appropriate here. We will go through the steps for doing this below.
```

Comment:
MoveIt! instead of moveIt,  "as CHOMP is not part of the official release yet. It is therefore not included in the binary packages."

Reply:
done